packages/@aws-cdk/aws-backup/lib/rule.ts

```diff
@@ -156,5 +173,26 @@ export class BackupPlanRule {
     if (props.scheduleExpression && !/^cron/.test(props.scheduleExpression.expressionString)) {
       throw new Error('`scheduleExpression` must be of type `cron`');
     }
+
+    let deleteAfter = props.deleteAfter;
+    if (props.enableContinuousBackup && !props.deleteAfter) {
+      deleteAfter = Duration.days(35);
+    }
+
+    if (props.enableContinuousBackup && props.moveToColdStorageAfter) {
+      throw new Error('`moveToColdStorageAfter` must not be specified if `enableContinuousBackup` is enabled');
+    }
+
+    if (props.enableContinuousBackup && props.deleteAfter &&
+      ((props.deleteAfter?.toSeconds() < Duration.days(1).toSeconds() ||
```

Comment:
Here I would prefer just `props.deleteAfter?.toDays() < 1 || props.deleteAfter?.toDays() > 35`

Reply:
I tried to change it. Now, the test case failed with message
`Received message: "'12 hours' cannot be converted into a whole number of days."`.
Should I change it anyway? I would prefer the existing solution because it shows a clear error message.

Reply:
This actually requires some more investigation. Can you confirm the granularity that `deleteAfter` allows? I know its called `deleteAfterDays` on cloudformation but that it also accepts a `double`. Which makes me think we are allowed fractions of days. On our side, we convert `deleteAfter` to days here (so we would run into the message you got anyway):

https://github.com/aws/aws-cdk/blob/d5362f001656dfeecc5d8388b9f1c9ce77190ea4/packages/%40aws-cdk/aws-backup/lib/plan.ts#L184

This could be a small bug, in that our checks are more stringent then cloudformation. 

If you're interested in checking this out, please do! If not, I like even more the conditional I suggested. It looks like the current CDK behavior is that `deleteAfter` must be a whole number of days, so it is good to stay with that level of granularity here.

Reply:
I changed it as proposed. I checked some other modules in CDK where `Duration` is used. It's implemented in the same way.
The granularity of `deleteAfter` is days. Message `"'12 hours' cannot be converted into a whole number of days."` indicates that developers have to specify `deleteAfter` in days if something else is used. If this is resolved, the logic checks if `deleteAfter` has a valid range. So it's fine for me. :)

Reply:
Perhaps I am misunderstanding you. But the `'12 hours...` error message comes from the line I linked above. It is a CDK limitation. I am weary of this because it seems like Cloudformation may allow fractions of days. This is good for now, I'll create an issue to investigate this further.